### ISSUE TITLE
Question about modded blocks 

### ISSUE THREAD

**Issue by TheBigSTN** [Querz/mcaselector#511]
Hello ! 
I don't think i can add the Question Tag ( i think moderators have too ) 
My Question is is there support for modded blocks like to get the block collor and not the black default like :
In game is : 
![image](https://github.com/user-attachments/assets/3dc041b7-6c74-47ad-9d3f-7bb005bf659a)
And in MCA : 
![image](https://github.com/user-attachments/assets/1b1a3c17-3fb2-4bd8-886d-7dffea4e66e1)
I am preety sure it dosen't support modded blocks but better be sure.

**Comment by Nervonment** [comment]
I think it is impossible to cover all kinds of modded blocks, since there are thousands of mods. But except for display, I think pasting chunks with modded blocks should work. Have you tried it?

**Comment by TheBigSTN** [comment]
I have not tried to paste chunks but i don't see why it won't be possible / work.
In the block color thing cant you read the mods folder for the top texture of all blocks and for every block combine the color so you get a color for every block ?
Like you get the leaf top texture from above and get the average of the pixels  to get one color reprezenting the block color.
I think that is how minecraft maps do it although i am not sure.
From what i understand the block colors are hardcoded ? Or are they discovered during loading or something ?

**Comment by TheBigSTN** [comment]
Yes it only reads the world folder but how hard would it be to read the mods folder for blocks 
I know that because when i tried to change the world of a server i needed to open the world folder and not the server folder
I think they are hardcoded i am refering to the colors
